let a = 23;
let b = 45;
let sum = a + b;
let avg = sum/2
console.log(sum)
try
    {
        console.log(x);
    }
catch(error)
    {
        console.log(error)
    }
finally {
    // Code that will always run
    console.log("This will run regardless of whether an error occurred.");
}
console.log(avg);